src/components/SingleDatePicker.jsx

```diff
@@ -218,6 +218,12 @@ export default class SingleDatePicker extends React.Component {
       values(visibleDays).forEach((days) => {
         Object.keys(days).forEach((day) => {
           const momentObj = moment(day);
+          if (isOutsideRange(momentObj)) {
```

Comment:
Gotchya... Thanks @majapw and @timhwang21, that makes sense. Let me go ahead and make that improvement.